wordpress-mobile/WordPress-iOS
# Cleanup unused unit test files from the *Remote migration to WPKit

This PR cleans up some duplicated files in the main WP app that were leftovers from the WordPressKit migration. Namely:

* Delete `RemoteTestCase.swift` from the main app
* Delete `XMLRPCTestable.swift` from the main app
* Move `RESTTestable.swift` to the WPKit framework

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.

@astralbodies would you mind reviewing this? Should be quick!
